Automattic/wp-calypso
# Cancellation form: Update styling

Related to #https://github.com/Automattic/wp-calypso/issues/93601

This PR updates the cancellation forms and the Atomic downgrade notices

## Proposed Changes

* Update the design for the Cancellation form survey
* Update the design for the Atomic downgrade
* Update the colors of various button upsells

## Why are these changes being made?


* Makes the forms more consistent with the overall design branding

## Testing Instructions



* Use the calypso live link
* Go to Purchases > Plan
* Click on the cancel subscription
* Compare it with the provided [designs](https://github.com/Automattic/wp-calypso/issues/93601#issuecomment-2358095968)
* Make sure it's fine on mobile resolutions
* The error notices behind the inputs were not added because it would have required more effort
* The disabled button was kept as it is

## Screenshots
| Before    | After |
| -------- | ------- |
| ![Screenshot 2024-11-01 at 15 54 46](https://github.com/user-attachments/assets/46a11ad9-d635-48b2-afd1-d3a9598dbc0e)   |   ![Screenshot 2024-11-01 at 16 02 48](https://github.com/user-attachments/assets/5152e50c-9b43-4f30-bda2-c64876178837) |
| ![Screenshot 2024-11-01 at 16 05 03](https://github.com/user-attachments/assets/758d2eba-760c-4019-a22a-620ae23a1c0d)  |   ![Screenshot 2024-11-01 at 16 03 50](https://github.com/user-attachments/assets/1726be66-45d5-4ce3-8d14-f73ecfea60f1) |
|   ![Screenshot 2024-11-01 at 16 06 33](https://github.com/user-attachments/assets/0daf796f-089d-4f26-bdba-4a65bc745b40)  |   ![Screenshot 2024-11-01 at 16 07 28](https://github.com/user-attachments/assets/7010ef0b-66f5-462a-8d82-ec9f17474c68) |
| ![Screenshot 2024-11-01 at 16 10 48](https://github.com/user-attachments/assets/cb3da76e-a0ee-4363-be8a-a2a5c19600b3) | ![Screenshot 2024-11-01 at 16 09 53](https://github.com/user-attachments/assets/a8dd630f-d831-4131-91cb-2fe09c6a79fb) |
## Pre-merge Checklist
